src/main/java/io/github/vhoyon/bot/commands/CommandClear.java

```diff
@@ -66,7 +67,44 @@ public void action(){
 		
 		boolean shouldDoClear = true;
 		
-		if(hasParameter("u", "s", "b")){
+		if(hasParameter("c")){
+			String content = getParameter("c").getContent();
+			
+			if(content == null){
+				addReplacement("Prefixes", code(getRequest().getCommandPrefix()));
```

Comment:
I would put the result of the potentially expensive `getRequest().getCommandPrefix()` in a variable and use it at your line 74 and your line 79. I'm thinking about optimization because CommandClear is relatively heavy already, might as well optimize as much as we can :)